package-lock.json

```diff
@@ -0,0 +1,2525 @@
+{
+  "name": "InfiniTime",
```

Comment:
SO MANY LINES! I have no idea how node works, but why do we need this huge file? Shouldn't it be enough to list the two needed packages?

Reply:
This file is the package lock, that specifies the versions of all the packages that are needed (which is more than just the 2 direct dependencies), so that builds are reproducible. It prevents npm pulling newer versions of packages, which might have changed things we (or other packages) rely on. The `package.json` specifies our direct dependencies.

Reply:
Reproducibility is good! Please document how this file is created, or is it always created by `npm install`?

Reply:
It's created and updated by any npm command that affects the `node_modules` directory or the `package.json`, so it is just created by running npm install. See https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json.